### ISSUE TITLE
Unable to save layout for use after Emacs reboot ?

### ISSUE THREAD

**Issue by dariuskramer** [seudut/perspeen#21]
Am I missing something or is there no way to save a configured layout for use after Emacs reboot ?
`desktop-save` does not work either :/

**Comment by machinekoder** [comment]
Yeah, I was expecting this feature too. It's nice to have workspaces, but without saving and restoring them it's pretty useless.
